MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+include CHANGELOG.md LICENSE README.md
+
+graft scripts
```

Comment:
I wouldn't include these, they're really for personal use so I don't have to remember commands.